cmd/minikube/cmd/start_flags.go

```diff
@@ -503,6 +510,27 @@ func getNetwork(driverName string) string {
 	return n
 }
 
+func validateVfkitNetwork(n string) string {
+	if runtime.GOOS != "darwin" {
+		exit.Message(reason.Usage, "The vfkit driver is only supported on macOS")
+	}
+	switch n {
+	case "nat":
```

Comment:
I believe by default the network value is empty, so would "nat" ever be chosen ?

Reply:
Yes, but if you use --network nat we will get "nat" here. If we document that we support "nat" it must be possible to use this value.